From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pavel Tumakaev <p.tumakaev@omprussia.ru>
Date: Tue, 2 Mar 2021 20:28:50 +0300
Subject: [PATCH] [sailfishos][gecko] Add missing #include for nsIObserver and
 startupObserver

This fixes "use of undeclared identifier" errors for nsIObserver and
startupObserver in nsAppStartupNotifier.cpp.
---
 toolkit/xre/nsAppStartupNotifier.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/toolkit/xre/nsAppStartupNotifier.cpp b/toolkit/xre/nsAppStartupNotifier.cpp
index f2a5ca59e05d..9ef7b9f5cb03 100644
--- a/toolkit/xre/nsAppStartupNotifier.cpp
+++ b/toolkit/xre/nsAppStartupNotifier.cpp
@@ -9,6 +9,7 @@
 #include "nsICategoryManager.h"
 #include "nsXPCOM.h"
 #include "nsAppStartupNotifier.h"
+#include "nsIObserver.h"
 #include "mozilla/SimpleEnumerator.h"
 
 /* static */
-- 
2.31.1

